lib/orbit/import/import_certifications.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Orbit.Import.ImportCertifications do
+  alias Orbit.Certification
+  alias Orbit.Import.ImportHelpers
+  alias Orbit.Repo
+
+  @spec import!(Enumerable.t(), Certification.certification_type()) :: :ok
+  def import!(file_lines, certification_type) do
+    file_lines
+    |> Enum.drop_while(fn line -> String.trim(line) != "" end)
+    |> Enum.drop_while(fn line -> String.trim(line) == "" end)
+    |> ImportHelpers.parse_csv(true)
+    |> Enum.map(fn certification ->
+      title = certification["Certifications - Certification Title"]
+      expires_string = certification["Certifications - Certification Period Expiration Date"]
+
+      expires =
+        expires_string |> Timex.parse!("{M}/{D}/{YYYY} {h12}:{m} {AM}") |> NaiveDateTime.to_date()
+
+      rail_line =
+        cond do
+          String.ends_with?(title, "BL") ->
+            :blue
+
+          String.ends_with?(title, "OL") ->
+            :orange
+
+          String.ends_with?(title, "RL") ->
+            :red
+
+          true ->
+            nil
```

Comment:
How do we want to handle lines we can't parse?
- Drop the row
- Crash 
- Save with a missing nil

Currently, this will save the nil, but certification.ex `@type t` doesn't say the field is nillable.

We probably also want to log a warning.

Reply:
Changed to be nillable; uncertain about the warning. We aren't generally logging these sorts of things, so they're not useful if we aren't looking for them. Are you thinking we would throw a splunk alert on it?

Reply:
Definitely not worth a splunk alert. I was thinking it might occasionally be useful if you see it locally or if doing a manual search for it while debugging an issue.

But I guess if we're manually debugging something, it's just as easy to grab the CSV from S3 and see the missing title there.

If we expect this to show up rarely or never, then might as well log it, that's what Log.warn is for, but it's definitely not necessary.